testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));
-
-    vmArgs.append(" ");
-    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));

```

Comment:
yep, i think i was pointing to the wrong testng version in pom.xml which was built with java 8.

anyway, looks like the plugin not including the patch:
```
============== VM args =============
-Dx=y
-ea
-Dx=y
```

could you have a check?